addons/a11y/src/index.js

```diff
@@ -6,22 +6,29 @@ import { logger } from '@storybook/client-logger';
 
 import { CHECK_EVENT_ID, REQUEST_CHECK_EVENT_ID } from './shared';
 
-let axeOptions = {};
+const config = {
+  axeOptions: {},
+  contextElementId: '',
+};
 
-export const configureA11y = (options = {}) => {
-  axeOptions = options;
+export const configureA11y = (axeOptions = {}, contextElementId = '') => {
+  config.axeOptions = axeOptions;
+  config.contextElementId = contextElementId;
 };
 
 const runA11yCheck = () => {
   const channel = addons.getChannel();
-  const infoWrapper = document.getElementById('story-root').children;
-  const wrapper = document.getElementById('root');
+
+  let axeContext = config.contextElementId;
```

Comment:
let's extract the `axeContext` discovery to a fnction getAxeContext() or something like this.